edwin/client/static/js/components/Timeline.js

```diff
@@ -24,22 +27,34 @@ import Icon from './Icon.js';
  */
 export default class Timeline extends ControllerComponent {
   get stores() {
-    return [BugStore, UserStore];
+    return [BugStore, UserStore, IndicatorStore];
   }
 
   loadData() {
     let teamSlug = this.props.params.team;
     let bugQuery = {comment_tag: `edwin-${teamSlug}`};
 
+    IndicatorActions.updateState(LoadingStates.USER_RESTORE);
```

Comment:
I think Ii may ditch the constant states and instead have each thing yield a "begin work x" "end work x" thing. So then every task has its own progress and we can do asynchronous stuff and all that.